src/main/java/kr/codesqaud/cafe/controller/MemberController.java

```diff
@@ -42,35 +44,45 @@ public String saveUser(@ModelAttribute("user") Member user) throws SQLException
 	}
 
 	@GetMapping("/list")
-	public String users(Model model) throws SQLException {
+	public String users(HttpSession session, Model model) throws SQLException {
+		if (session.getAttribute(SESSIONED_USER) == null) {
+			return "user/login";
+		}
 		List<Member> members = namedJdbcTemplateMemberRepository.showAllUsers();
 		model.addAttribute("users", members);
 		log.trace("사용자 수: {}", members.size());
 		return "user/list";
 	}
 
-	@GetMapping("/profile/{userId}")
-	public String userProfile(@PathVariable String userId, Model model) throws SQLException {
-		Optional<Member> findUser = namedJdbcTemplateMemberRepository.findById(userId);
+	@GetMapping("/profile/{userSequence}")
+	public String userProfile(HttpSession session, @PathVariable Long userSequence, Model model) throws SQLException {
```

Comment:
```suggestion
	public String getProfile(HttpSession session, @PathVariable Long userSequence, Model model) throws SQLException {
```

콘트롤러 메소드의 네이밍 컨벤션도 한 번 공부해 보시면 좋을 것 같아요.

Reply:
넵 알겠습니다